example/20-componentconfig-gardenlet.yaml

```diff
@@ -40,6 +40,7 @@ controllers:
   shootCare:
     concurrentSyncs: 5
     syncPeriod: 30s
+    healthCheckOutdatedThreshold: 5m
```

Comment:
Maybe it should be called `staleExtensionHealthCheckThreshold` to make it more clear that it's about the extension health checks?

Reply:
Can also adapt the `gardener/controlplane` chart if necessary?

Reply:
you mean gardener/gardenlet? -> done